ejb-security-context-propagation/src/main/java/org/jboss/as/quickstarts/ejb_security_context_propagation/IntermediateEJB.java

```diff
@@ -36,7 +36,7 @@
 @PermitAll
 public class IntermediateEJB implements IntermediateEJBRemote {
 
-    @EJB(lookup="ejb:/ejb-security-context-propagation/SecuredEJB!org.jboss.as.quickstarts.ejb_security_context_propagation.SecuredEJBRemote")
+    @EJB
```

Comment:
nowadays you just EJB to inject, unless it's not resolvable of course, this abstract us from the deployment name too (which is ejb-security-context-propagation for standard dist, but ROOT for server provisioining)